appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockListView.java

```diff
@@ -382,6 +387,17 @@ public void onPropertyChange(String propertyName, String newValue) {
         setElementsFromStringProperty(currentElements);
       }
       refreshForm();
+    } else if (propertyName.equals(PROPERTY_NAME_TEXTSIZE)) {
+      EditableProperty fontSize = properties.getProperty(PROPERTY_NAME_FONTSIZE);
+      fontSize.setValue(newValue);
+    } else if (propertyName.equals(PROPERTY_NAME_FONTSIZE)) {
+      EditableProperty textSize = properties.getProperty(PROPERTY_NAME_TEXTSIZE);
+      if (textSize != null) {   // This will be null while project is loading
+                                // because FontSize is set before TextSize exists
+        float fontSize = Float.parseFloat(newValue); // Need to make sure it is an integer
+        int intTextSize = Math.round(fontSize);
+        textSize.setValue(String.valueOf(intTextSize));
+      }
```

Comment:
I propose that we should hide the `FontSize` property for this release and then in a future release (it can even be master since the companion will have the code) we can provide the upgrader to transition from TextSize to FontSize.

Reply:
@ewpatton By hide, do you mean remove?

Reply:
@ewpatton Or do you mean remove it as a designer property and hide the block, but leave in the support in the component so that the Companion knows about it?

Reply:
The latter. Remove the annotations but leave the function implementations. For `@SimpleProperty` we can even set the `userVisible` flag to false to hide it in the blocks (doesn't affect the designer though).

Reply:
So the changes to MockListView have been flushed.